locales/GOOGLE_en.json

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
while it is funny, please remove it from the pr, I cant merge it now 😄

Reply:
Done